docs/roles/system_manager/manage_users.md

```diff
@@ -64,9 +66,11 @@ Once you're certain that you're adding a new user, make sure that the following
 - Be particularly careful never to use them to log in to any user-accessible VMs (such as the SRDs)
 ```
 
+(adding_users_manually)=
+
 ### {{woman}} {{man}} Adding users to security group(s)
 
-Users that didn't have a `GroupName` specified in the `user_details_template.csv`, will need to be added manually, before they can access an SRE.
+Users will need to be added to the relevant security group before they can access an SRE. This may have already been done if the `GroupName` was specified in the `user_details_template.csv` (see {ref}`generate_user_csv`), but otherwise can be achieved by the following:
```

Comment:
Again, this feels like it is in the wrong place.

This section is about adding a user to a group, for any purpose. I think the warning about users not being able to access the SRE if they are in no group would be better placed with the creating users/users csv. That way we might help people avoid that mistake. If you are reading about it here, you have probably already messed up.